package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@zeit/dockerignore",
+  "name": "@balena/dockerignore",
   "version": "0.0.5",
-  "description": "dockerignore is a manager and filter for .dockerignore rules and is a fork of ignore.",
+  "description": "dockerignore is a file filter library compatible with Docker and the node-ignore API",
```

Comment:
Reasoning:
* This package's name is already `dockerignore`, so 2 further occurrences of `dockerignore` in the description line seemed redundant.
* While I can understand the meaning of "manager and filter" given all the context I've got, I thought it was rather vague/generic to a developer doing a package search -- especially the "manager" term.
* The good thing about the expression "a fork of ignore" is that it clarifies that "ignore" is a repo (although the github repo is actually called "node-ignore"), but I thought it didn't answer the question _"so what?"_ ("a fork of ignore" - ok, so what?...) Many users won't have knowledge of the "ignore" package, even though it's popular on `npm`.
* My proposed description is not perfect either, but I thought the advantages were:
  * It includes the words "Docker" and "node", which are widely known.
  * "compatible with the node-ignore API" hints an answer to the "so what" question: API compatibility. Granted, people won't know what "the node-ignore API" is, but:
    * They will be left with the impression that "node-ignore API" is a thing they should know about.
    * If they google "node-ignore", they get the right answers as the top 3 results! Whereas if they google just "ignore," they get too much "noise".
  * Experienced Node.js developer will recognise the pattern `node-xxx` as the Github repo corresponding to the `npm` `xxx` package, hence `node-ignore` also hints at an `npm` package called `ignore`.